### ISSUE TITLE
Is there a way to output >= instead of == in requirements.txt?

### ISSUE THREAD

**Issue by philipperemy** [bndr/pipreqs#117]
Is there a way to output >= instead of == in requirements.txt?

```
pygsheets>=1.1.3
retrying>=1.3.3
```

Instead of 

```
pygsheets==1.1.3
retrying==1.3.3
```

Thanks!

**Comment by Honghe** [comment]
Or how to export without specific version number?

**Comment by philipperemy** [comment]
@Honghe that works too!

**Comment by sirfoga** [comment]
@philipperemy use `sed`:
```bash
pipreqs --force --print . | sed "s|==|>=|" > requirements.txt
```

**Comment by philipperemy** [comment]
@sirfoga Thanks! Works great. Nice hack :)

**Comment by rklasen** [comment]
@sirfoga thats a good workaround, but is there a flag for pipreqs to export without any version?

**Comment by sirfoga** [comment]
@rklasen not that I know of. But you can always use sed:
```
pipreqs --force --print . | sed -e "s|==.*||"
```
will replace `==` and everything after that, with nothing

```
pandas
httplib2
scipy
requests
setuptools
psutil
Send2Trash
numpy
stem
pymongo
mutagen
matplotlib
...
```

**Comment by alan-barzilay** [comment]
Feature implemented in PR#239

**Comment by philipperemy** [comment]
@alan-barzilay thank you!
